docs/cli.md

```diff
@@ -1,6 +1,6 @@
 # CLI Commands
 
-The following sections describe the CLI commands available to bot developers. You can always use `forta-agent help` to get a quick overview of these commands, as well as details about specific commands e.g. `forta-agent run help`.
+The following sections describe the CLI commands available to bot developers. You can always use `forta-bot help` to get a quick overview of these commands, as well as details about specific commands e.g. `forta-bot run help`. The command `forta-agent` is still supported and can be used interchangeably with `forta-bot`. 
```

Comment:
lets not mention the `forta-agent` support here. we'll tell people when announcing, but the docs should encourage use of `forta-bot`